paper.md

```diff
@@ -1,53 +1,97 @@
 ---
-title: 'pyuvdata: an interface for astronomical interferometeric datasets in python'
+title: 'pyuvdata v3: an interface for astronomical interferometeric datasets in python'
 tags:
  - radio astronomy
  - uvfits
  - miriad
 authors:
+ - name: Garrett K. Keating
+   orcid: 0000-0002-3490-146X
+   affiliation: 1
  - name: Bryna J. Hazelton
    orcid: 0000-0001-7532-645X
-   affiliation: 1, 2
+   affiliation: 2, 3
+ - name: Matthew Kolopanis
+   orcid: 0000-0002-2950-2974
+   affiliation: 4
+ - name: Steven Murray
+   orcid: 0000-0003-3059-3823
+   affiliation: 4
```

Comment:
My affiliation is now `Scuola Normale Superiore, Italy`

Reply:
Fixed!